### ISSUE TITLE
It stops working after a few seconds

### ISSUE THREAD

**Issue by dimii27** [ratwithacompiler/OBS-captions-plugin#62]
It actually works for up to one minute but then it just doesn't generate new captions, even when played the same recording again. I'm using it when recording. Maybe it is an issue with it trying to recognise a tough word and gets stuck?

**Comment by jmytch** [comment]
Yeah I noticed this has been happening to some streamer on Twitch even with English language. Looks like it needs updating, not sure what made it not work for 10 minutes while it is active the whole time. Used to work fine in previous versions

**Comment by dimii27** [comment]
> Yeah I noticed this has been happening to some streamer on Twitch even with English language. Looks like it needs updating, not sure what made it not work for 10 minutes while it is active the whole time. Used to work fine in previous versions

How do I uninstall the plugin to install an older version?
